stemcell_builder/stages/password_policies/apply.sh

```
@@ -33,6 +33,18 @@ patch -p1 $chroot/etc/pam.d/common-auth < $assets_dir/ubuntu/common-auth.patch
 strip_trailing_whitespace_from $chroot/etc/pam.d/common-password
 patch -p1 $chroot/etc/pam.d/common-password < $assets_dir/ubuntu/common-password.patch
 
+# libpam-lastlog2 installs pam_lastlog2.so only to the multiarch path
+# (e.g. /usr/lib/x86_64-linux-gnu/security/), but PAM's securedir is
+# /usr/lib/security/ -- without this systemd logs:
+#   PAM unable to dlopen(pam_lastlog2.so): /usr/lib/security/pam_lastlog2.so:
+#   cannot open shared object file
+for module in $chroot/usr/lib/*-linux-gnu/security/pam_lastlog2.so; do
```

Shellcheck (0.11.0)

[info] Quote expansions in this for loop glob to prevent wordsplitting, e.g. "$dir"/*.txt .

(SC2231)